src/moscot/problems/space/_alignment.py

```diff
@@ -33,9 +33,24 @@ def prepare(
         policy: Literal["sequential", "star"] = "sequential",
         reference: Optional[str] = None,
         cost: Union[
-            Literal["sq_euclidean", "cosine", "bures", "unbalanced_bures"],
-            Mapping[str, Literal["sq_euclidean", "cosine", "bures", "unbalanced_bures"]],
+            Literal[
+                "euclidean", "sq_euclidean", "cosine", "pnorm_p", "sq_pnorm", "sq_pnorm", "elastic_l1", "elastic_stvs"
+            ],
+            Mapping[
+                str,
+                Literal[
+                    "euclidean",
```

Comment:
Same, please add an alias, won't mention later.

Reply:
didn't do it for documentation readability, but happy to adapt.